velox/dwio/common/compression/Compression.cpp

```diff
@@ -245,9 +246,10 @@ uint64_t LzoAndLz4DecompressorCommon::decompress(
     DWIO_ENSURE_GE(
         compressedSize,
         dwio::common::INT_BYTE_SIZE,
-        "{} decompression failed, input len is too small: {}",
-        kind_,
-        compressedSize);
+        fmt::format(
+            "{} decompression failed, input len is too small: {}",
+            kind_,
```

Comment:
Please replace all occurrences of `kind_` w/ `::facebook::velox::common::compressionKindToString(kind_)`, instead of `kind_`